test/e2e/windows/README.md

```diff
@@ -0,0 +1,10 @@
+# Notes to run sig-windows tests
+
+1. Prereqs:
+    * KUBECONFIG=path/to/kubeconfig
+    * curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/repo-list.txt -o repo_list.yaml
```

Comment:
This should be updated to using the official sig-windows testing repo soon.

Reply:
Thanks for the review @yujuhong. Which repo are you referring to?

Reply:
This one, and I filed an issue: https://github.com/kubernetes-sigs/windows-testing/issues/1

@PatrickLang @adelina-t could we move the shared e2e utilities to the `windows-testing` repo, while waiting for the e2e test tagging to mature?

Reply:
This URL changed - I updated it in https://github.com/PatrickLang/kubernetes/commit/254b3348e77836cd1e4e56ab81c27f229d030dea

I think we should move it to the windows-testing repo, but not before this change. I think we need to get an automatic build running there first and the repo list should point to the output of that build.